www-servers/tornado/tornado-6.1.ebuild

```diff
@@ -32,6 +32,14 @@ distutils_enable_sphinx docs \
 	dev-python/sphinx_rtd_theme \
 	dev-python/sphinxcontrib-asyncio
 
+python_prepare_all() {
+	# Disable deprecation-warnings-as-errors because tornado has a lot of stuff deprecated in 3.10
+	sed 's/warnings.filterwarnings("error", category=DeprecationWarning, module=r"tornado\\..*")//' \
```

Comment:
This isn't going to fly. The warnings come from tornado itself, not the tests. You're shifting the problem to another package that will detect the warnings and fail.

Reply:
I'll look into fixing it in the package itself and submitting it upstream

Reply:
Seems like this one is too complicated for me, unfortunately, there are _a lot_ of deprecated things used.

Reply:
Seems like this one's too hard for me, there's too much deprecated in there that I don't understand. I'll try at least opening an issue upstream about this.

Reply:
Could you make a PR with at least these that you know how to fix? I think I've seen TLS-related stuff somewhere (urllib3? requests?), so you may want to try looking for a solution there.

Reply:
When i try to fix things unfortunately it breaks even more :/ 
Unfortunately this one might be fit more for a real python developer

Reply:
Got a response on the issue, they said it's probably going to be a major pain for them because of the get_current_loop deprecation.